### PR TITLE
Switch to actions workflow

### PR DESCRIPTION
**Pull request by montgomeryyoung**
## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes/no
| Did you apply meaningful labels to the pull request? | yes/no
| Did you perform a self review first? | yes/no
| Documentation reflects changes? | yes/no
| `CHANGELOG` reflects changes? | yes/no
| Unit/Functional tests reflect changes? | yes/no
| Did you perform browser testing? | yes/no
| Risk level | Low, Medium, High
| Relevant links | JIRA issue, bug reports, security alerts, etc.

### CODE REVIEW

File: .github/workflows/build-and-deploy.yml [hunk 1]
```diff
@@ -7,98 +7,27 @@ on:
       - '**'
     tags:
       - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+-rc'
```
Comment (montgomeryyoung) [inline .github/workflows/build-and-deploy.yml:10]: I am not sure if we can use the same tags as the other project

File: .github/workflows/build-and-deploy.yml [hunk 2]
```diff
@@ -7,98 +7,27 @@ on:
...
-        with:
-          name: build.tar.gz
-          path: /tmp/build.tar.gz
-
-  # Deploy all branches to Acquia
-  branch_deploy:
-    name: Deploy environment branches
+    uses: oomphinc/oomphcloud/.github/workflows/build-drupal.yml@main
+    with:
+      node-version: '10.x'
```
Comment (montgomeryyoung) [inline .github/workflows/build-and-deploy.yml:16]: I ran `lando node -v` in the project and got 10.24.1

File: .github/workflows/build-and-deploy.yml [hunk 3]
```diff
@@ -7,98 +7,27 @@ on:
...
-      - name: Expand the build artifact
-        run: tar -xzf /tmp/build.tar.gz
-
-      - name: Deploy to Acquia
-        env:
-          ACQUIA_SSH_KEY: ${{ secrets.ACQUIA_SSH_KEY }}
-        run: ./oomph-infrastructure/oomph-action deploy-acquia.yml -e "acquia_target_branch=master"
+    uses: oomphinc/oomphcloud/.github/workflows/deploy-drupal-acquia.yml@main
+    with:
+      acquia-repo-hostname: 'svn-21939.prod.hosting.acquia.com'
```
Comment (montgomeryyoung) [inline .github/workflows/build-and-deploy.yml:27]: I don't know these 3 variable values...so I temporarily used the values from another project.

**Comment by edaemon** [comment]
This has been superseded by #306.

**Pull request closed without merging**